.github/workflows/links.yml

```
@@ -0,0 +1,40 @@
+name: Links (Fail Fast)
```

```suggestion
name: Check external links
```

blocking -- this is a clearer description of this workflow.